api/graphql/schema.py

```diff
@@ -197,7 +197,7 @@ def resolve_queryset(
                 application_events__event_reservation_units__reservation_unit__unit__in=units
             )
             | Q(user=user)
-        ).distinct()
+        )
```

Comment:
Could you clarify why `distinct` is not needed here anymore? Just wanting to make sure it isn't an accidental removal 😄

Reply:
I certainly can't! :D So very nice catch 🎣

Reply:
fix'd